packages/monaco/src/twoslash/queries.ts

```diff
@@ -53,7 +53,7 @@ export async function checkTwoslashQuery({
       offset: inspectionOff,
       worker,
     });
-    const inlayHint = await createInlayHint({
+    const inlayHint = createInlayHint({
```

Comment:
[`@typescript-eslint/await-thenable`](https://typescript-eslint.io/rules/await-thenable): because `createInlayHint` was switched from `async` to synchronous, this `await` is no longer necessary. 🔪